tests/buildTools/src/test/scala/tests/BaseBuildToolSuite.scala

```diff
@@ -130,3 +148,56 @@ abstract class BaseBuildToolSuite extends MopedSuite(ScipJava.app) {
   }
 
 }
+
+object BaseBuildToolSuite {
+  lazy val externalJavaVersion: Int = {
+    val tmpDir = os.temp.dir()
+    var version = Option.empty[String]
+    try {
+      os.write(tmpDir / "PrintJavaVersion.java", PrintJavaVersion)
+
+      os.proc("javac", "PrintJavaVersion.java").call(cwd = tmpDir)
+
+      version = Some(
+        os.proc("java", "PrintJavaVersion").call(cwd = tmpDir).out.text()
+      )
+    } finally {
+      os.remove.all(tmpDir)
+    }
+
+    version
+      .map(parseJavaVersion)
+      .getOrElse(sys.error("Failed to detect external JDK version"))
```

Comment:
It's fine to hard die here, because an external JVM _has_ to be available for all the build tool tests (gradle, maven, sbt won't start without it)